video_prediction/prediction_model.py

```diff
@@ -172,25 +173,26 @@ def construct_model(images,
       enc6 = slim.layers.conv2d_transpose(
           hidden7,
           hidden7.get_shape()[3], 3, stride=2, scope='convt3',
+          activation_fn=None,
           normalizer_fn=tf_layers.layer_norm,
           normalizer_params={'scope': 'layer_norm9'})
 
       if dna:
         # Using largest hidden state for predicting untied conv kernels.
         enc7 = slim.layers.conv2d_transpose(
-            enc6, DNA_KERN_SIZE**2, 1, stride=1, scope='convt4')
+            enc6, DNA_KERN_SIZE**2, 1, stride=1, scope='convt4', activation_fn=None)
       else:
         # Using largest hidden state for predicting a new image layer.
         enc7 = slim.layers.conv2d_transpose(
-            enc6, color_channels, 1, stride=1, scope='convt4')
+            enc6, color_channels, 1, stride=1, scope='convt4', activation_fn=None)
         # This allows the network to also generate one image from scratch,
         # which is useful when regions of the image become unoccluded.
         transformed = [tf.nn.sigmoid(enc7)]
 
       if stp:
         stp_input0 = tf.reshape(hidden5, [int(batch_size), -1])
         stp_input1 = slim.layers.fully_connected(
-            stp_input0, 100, scope='fc_stp')
+            stp_input0, 100, scope='fc_stp', activation_fn=None)
```

Comment:
Thank you! I've change this line back to the original where the ReLU activation was default.